src/nft/components/collection/FilterButton.tsx

```diff
@@ -17,15 +17,9 @@ export const FilterButton = ({
   onClick: () => void
   collectionCount?: number
 }) => {
-  const isCollectionNftsLoading = useIsCollectionLoading((state) => state.isCollectionNftsLoading)
-
   return (
     <Box
-      className={
-        isCollectionNftsLoading
-          ? styles.filterButtonLoading
-          : clsx(styles.filterButton, !isFiltersExpanded && styles.filterButtonExpanded)
-      }
+      className={clsx(styles.filterButton, !isFiltersExpanded && styles.filterButtonExpanded)}
```

Comment:
this was requested to remove a while ago.  This button only shows a loading state on initial load now